src/select.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+import JWSelect from 'react-select';
+import classNames from 'classnames';
+import identity from './helpers/identity';
+
+export default class Select extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: props.defaultValue
```

Comment:
Hard coded values are less useful, it gets interesting when we have variables set as `value`. These controlled inputs give us ways to enforce what value an input displays, such as formatting text, or restricting things. 

```
React.createClass({
    getInitialState() {
        return { value: '' };
    },
    onChange(e) {
        this.setState({ 
            value: e.target.value.slice(0, 5)
        });
    },
    render() {
        return <input 
            onChange={this.onChange}
            value={this.state.value} />
    }
});
```
Here we have an input where we limit the length to 5 characters long by always rendering a `value` in the input. 

For reference:
https://facebook.github.io/react/docs/forms.html#why-controlled-components